src/services/activation/services/ActivationService.js

```diff
@@ -0,0 +1,89 @@
+const { authenticate } = require('@feathersjs/authentication').hooks;
+const logger = require('../../../logger');
+
+const {
+	lookupByUserId,
+	deleteEntry,
+	filterEntryParamNames,
+	validEntry,
+	getUser,
+	lookupByActivationCode,
+	NotFound,
+	customErrorMessages,
+} = require('../utils');
+
+// This service does all general things like finding open jobs,
+// redeeming activation codes, deleting jobs. But this service
+// does not deal with what should happen when the activation
+// code is redeemed, e.g. changing the email/user name.
+class ActivationService {
+	// find open jobs
+	// will return filterd array of open jobs
+	async find(params) {
+		const { userId } = params.account;
+		const entry = await lookupByUserId(this, userId);
```

Comment:
```suggestion
		const jobs = await lookupActivationJobsByUserId(this, userId);
```
use either job or entry, job sounds better for me

Reply:
job would probably be the better identifier, but entry is used 98% of the time, so changing it now takes time and can lead to errors. Therefore I now use entry

Reply:
i have renamed the function